src/rpc/blockchain.cpp

```diff
@@ -1274,7 +1274,7 @@ static RPCHelpMan gettxout()
                     {RPCResult::Type::STR, "asm", ""},
                     {RPCResult::Type::STR_HEX, "hex", ""},
                     {RPCResult::Type::STR, "type", "The type, eg pubkeyhash"},
-                    {RPCResult::Type::STR, "address", /* optional */ true, "The Bitcoin address (only if a well-defined address exists)"},
+                    {RPCResult::Type::STR, "address", /* optional */ true, DESTINATION_ADDRESS},
```

Comment:
it might be better to de-duplicate the whole asm/hex/type/address blob. Though, maybe create a separate pull for doc cleanups vs code refactorings?

Reply:
See for example e7507f333b or f8911de619d41a3cc21771d62ab077713f1405c5 on how to do this.

Reply:
I think this is a good suggestion. I will do this as a followup as you suggest. So I will get rid of 63a657ccff959f7d86f423607f69aa9f7507dbdb when I rebase and have a doc cleanup followup with these suggestions